docs/arcaflow/workflows/flow-control.md

```diff
@@ -153,16 +162,21 @@ There are two tags that create the described OR dependency.
 
 ##### How to use `!oneof`
 
-To use `!oneof` for graceful handling of disabled steps, the oneof should depend on ONE OF the success output and
-the disabled output, which will output either the success output or the disabled output.
+To use `!oneof` for graceful handling of disabled steps, the oneof should depend on ONE
+OF the success output and the disabled output, which will output either the success
+output or the disabled output.
 
-The oneof tag is a method of creating a schema `one_of_string` type from values present in the workflow.
+The oneof tag is a method of creating a schema `one_of_string` type from values present
```

Comment:
I think the "oneof" references here should probably all be rendered as `oneof` so the eye isn't expecting English.

Reply:
Yeah, good point. This was missed in the prior PR. This section was only changed in width in this PR.